app/templates/views/roadmap.html

```diff
@@ -15,83 +15,60 @@ <h1 class="heading-large">Roadmap</h1>
     }
   ) }}
 
 
   <p class="govuk-body">The GOV.UK Notify roadmap shows what we’re working on and some of the <a class="govuk-link govuk-link--no-visited-state" href="#things-we-have-done">things we’ve done</a>.</p>
   <p class="govuk-body">The roadmap is only a guide. It does not cover everything we do, and some things may change.</p>
-  <p class="govuk-body">You can <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('.support')}}">contact us</a> if you have any questions about the roadmap or suggestions for new features.</p>
+  <p class="govuk-body">You can <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('.support')}}">contact us</a> if you have any questions about the roadmap, want to get involved in user research or have suggestions for new features.</p>
 
   <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working on now</h2>
 
   <ul class="govuk-list govuk-list--bullet">
 
-    <li>Let services upload their own logo for emails and letters</li>
-    <li>Start to <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">migrate our database from GOV.UK PaaS</a> to one hosted using <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/rds/">Amazon RDS</a></li>
-    <li>Migrate one of our applications from GOV.UK PaaS to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
-    <li>Make it quicker and easier to turn off <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.trial_mode')}}">trial mode restrictions</a></li>
-    <li>Let services choose their own two-factor authentication method</li>
-    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_receive_text_messages') }}">receive text messages</a> without having to contact the Notify team first</li>
-    <li>Make it easier to find information and guidance on the Notify website</li>
-    <li>Design reusable content patterns for text messages</li>
-    <li>Update the version of the <a class="govuk-link govuk-link--no-visited-state" href="https://design-system.service.gov.uk/">GOV.UK Design System</a> that we use</li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrating our database and applications from GOV.UK PaaS</a> to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
+    <li>Let services create letters in the Welsh language to comply with the Welsh Language Act 1993</li>
+    <li>Add attachments to letters</li>
+    <li>Improve our email functionality so users can add images, QR codes and have more formatting options</li>
+    <li>Make it easier for services to request to go-live</li>
```

Comment:
Not sure this needs a hyphen

```suggestion
    <li>Make it easier for services to request to go live</li>
```